Vagrantfile

```diff
@@ -47,6 +47,13 @@ Vagrant.configure(2) do |config|
     vb.cpus = 2
   end
 
+  host_user = ENV.fetch("USER", "vagrant")
+  aws_profile = ENV.fetch("RF_AWS_PROFILE", "raster-foundry")
+  rf_aws_config_bucket = ENV.fetch("RF_AWS_CONFIG_BUCKET",
```

Comment:
Does it make sense to keep this named as `RF_SETTINGS_BUCKET` similar to the environment variable that's used in other places, but with the staging bucket reference?